lib/grpc_web/rack_handler.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'google/protobuf'
```

Comment:
Which part requires this? (couldn't grasp from the diff)

Reply:
It's not added in this PR, but the rescue on line 33 uses this namespace:
`rescue Google::Protobuf::ParseError => e`
It wasn't an issue before probably because it is loaded by another file already

Reply:
ahh! missed that. thanks